content/en/docs/migrating/_index.md

```diff
@@ -899,6 +899,96 @@ to wait for a {{% alias service=S3 %}} Bucket to exist, you must construct a `Bu
 [s3.BucketExistsWaiter]({{< apiref "service/s3#BucketExistsWaiter" >}}). The `s3.BucketExistsWaiter` provides a
 `Wait` method which can be used to wait for a bucket to become available.
 
+### Presigned Requests
+
+The V1 SDK technically supported presigning _any_ AWS SDK operation, however,
+this does not accurately represent what is actually supported at the service
+level (and in reality most AWS service operations do not support presigning).
+
+{{% alias sdk-go %}} resolves this by exposing specific `PresignClient`
+implementations in service packages with specific APIs for supported
+presignable operations.
```

Comment:
i dont think we actually have `PresignClient`s for every service that supports it. this is because in the past weve gotten feature requests to add a `PresignClient` for a specific operation, which we then fulfilled (i think this is the first task that Sean gave me to do https://github.com/aws/aws-sdk-go-v2/pull/1888).

if this is indeed the case, maybe we should add something about filing a feature request if they dont see a `PresignClient` for a service they know supports it?